### ISSUE TITLE
ZWAVE TILT SENSOR not connected to EcoLink App

### ISSUE THREAD

**Issue by DWK-007** [koktail-nl/com.discoverecolink.issues#1]
Hi - 

I recently upgraded my old ZWave Hub to Homey Pro 2023.  I see that the EcoLink App for Homey Pro 2023 is now available since 26 Sept 2023. 

I have two Ecolink ZWave Tilt Sensors (Model:  TILT-ZWAVE2).  I tried to connect one of them to Home Pro 2023 using your app.

However, after trying to trying to pair the sensor with the EcoLink App, Homey Pro 2023 gave a message that no app was found for this device. 

From the Homey Pro 2023 settings, the details for this device are: 
     Manufacturer ID:   330
     Product Type ID:       1
     Product ID:                3

If possible, please include the Ecolink ZWave Tilt Sensors (Model:  TILT-ZWAVE2) to the Ecolink App. 

Thank you!

**Comment by donkrobertson** [comment]
I also would appreciate support for the TILT-ZWAVE2.5-ECO for Homey Pro, Sold on Amazon US. I have a couple of these and they have been very reliable on previous hubs.

Manufacturer ID: 330
Product Type ID: 4
Product ID: 3

Regards.

**Comment by pathed1969** [comment]
+1 for support for the TILT-ZWAVE2 device...

**Comment by squizzeak** [comment]
I have a door/window sensor the does the same thing; "no app installed" despite adding the device from the ecolink app's supported devices.

**Comment by DWK-007** [comment]
Hi Leendert de Kok, 

Please provide any update, even if the answer is that these older sensors will not be supported. 

Thank you!

**Comment by grantlutz** [comment]
+1 for supporting tilt zwave2

**Comment by voghali** [comment]
Is this forum being monitored? This functionality was requested about 6 months ago and still no response on this. Shall I return the garage tilt sensors?

**Comment by trenchtownrock** [comment]
Yes, please add the tilt sensor to the Homey Pro app! Ecolink tilt sensors last longer and are more reliable than the other options out there.
